portal-api/src/v1/gef/routes.js

```diff
@@ -1,34 +1,31 @@
 const express = require('express');
 const { validate } = require('../../role-validator');
 const mandatoryCriteriaVersioned = require('./controllers/mandatoryCriteriaVersioned.controller');
+const application = require('./controllers/application.controller');
 
 const router = express.Router();
 
+// Application
+router.route('/application')
+  .get(application.getAll)
```

Comment:
Can we add a user validation on the GET requests too
e.g.  .get(
    validate({ role: ['maker', 'checker'] }),
    deals.findOne,
  )

Reply:
updated